Source/FikaAmazonAPI/Services/FeedService.cs

```diff
@@ -127,6 +127,17 @@ public CreateFeedDocumentResult CreateFeedDocument(ContentType contentType)
         /// <param name="feedType"></param>
         /// <returns></returns>
         public string SubmitFeed(string xml, FeedType feedType, FeedOptions feedOptions = null)
+        {
+            return SubmitFeed(xml, feedType, new List<string> { MarketPlace.ID }, feedOptions);
+        }
+
+        /// <summary>
+        /// read full step  https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/feeds-api-use-case-guide/feeds-api-use-case-guide_2021-06-30.md
+        /// </summary>
+        /// <param name="xml"></param>
+        /// <param name="feedType"></param>
+        /// <returns></returns>
+        public string SubmitFeed(string xml, FeedType feedType, List<string> marketPlaceIds, FeedOptions feedOptions = null)
```

Comment:
Can you change this as optional , and the default system will add current marketplace id if parameter null as you did in last pull request

Reply:
List<string> marketPlaceIds